aliyun/src/test/java/org/apache/iceberg/aliyun/oss/AliyunTestUtility.java

```diff
@@ -56,4 +61,28 @@ public static AliyunOSSTestRule initialize() {
 
     return testRule;
   }
+
+  public static String testOSSEndpoint() {
+    return systemEnv(ALIYUN_TEST_OSS_ENDPOINT);
+  }
+
+  public static String testAccessKeyId() {
+    return systemEnv(ALIYUN_TEST_ACCESS_KEY_ID);
+  }
+
+  public static String testAccessKeySecret() {
+    return systemEnv(ALIYUN_TEST_ACCESS_KEY_SECRET);
+  }
+
+  public static String testBucketName() {
+    return systemEnv(ALIYUN_TEST_BUCKET_NAME);
+  }
+
+  public static String testOssKeyPrefix() {
+    return systemEnv(ALIYUN_TEST_OSS_KEY_PREFIX);
+  }
+
+  private static String systemEnv(String name) {
```

Comment:
nit: seems a bit unnecessary to have an additional method just to wrap `System.getenv` and perform no additional action.

Reply:
+1 for @jackye1995 's comment .